src/components/Editor/FreeBusy/FreeBusy.vue

```diff
@@ -28,32 +28,104 @@
 			<div v-if="loadingIndicator" class="loading-indicator">
 				<div class="icon-loading" />
 			</div>
+			<div class="modal__content__header">
+				<h2>{{ t('calendar','Find a time') }}</h2>
```

Comment:
```suggestion
				<h2>{{ t('calendar', 'Find a time') }}</h2>
```

we need better eslint config